$NetBSD: patch-ad,v 1.1.1.1 2006/07/25 16:28:47 salo Exp $

lint complains so get rid of it, and fix
path for LIBDIR to work in pkgsrc

--- parser/Makefile.orig	2004-03-12 22:16:59.000000000 +0000
+++ parser/Makefile
@@ -2,13 +2,13 @@
 # $FreeBSD$
 
 LIB=		parser
-DESTDIR=
-LIBDIR=		.
+LIBDIR=		${PREFIX}/lib
 MAN=
 SRCS=		bnep.c cmtp.c hidp.c l2cap.c rfcomm.c hci.c parser.c sdp.c
 WARNS?=		2
 DEBUG_FLAGS=	-g
 CFLAGS=		-O2
+NOLINT=		1
 
 NOPROFILE=	1
 
